### DISCUSSION TITLE
Sound controls using HiFiberry DAC+ STANDARD

### DISCUSSION THREAD

**Discussion by MarkAsRead** [MiczFlor/RPi-Jukebox-RFID#1327] [Q&A]
Hi all,

I've been playing around with the Phoniebox for a while now and I really like it so far.

However I keep running into the following issue:

When I install Raspberry Pi OS with desktop Release date: January 11th 2021 and use the settings as described https://www.hifiberry.com/docs/software/configuring-linux-3-18-x to activate the HiFiberry the sound controls don't work.

In the install script I select the MPD as default and the Headphones.

Any help would be very much appreciated.

**Comment by lj2000** [comment]
Hi MarkAsRead,

follow the instructions on the following page under the section '**Hifiberry MiniAmp einrichten**':
[https://splittscheid.de/phoniebox-bauanleitung-toniebox-alternative/](https://splittscheid.de/phoniebox-bauanleitung-toniebox-alternative/)

Depending on your Hifiberry (DAC+) you must change the dtoverlay setting:
- For Miniamp: dtoverlay=hifiberry-dac
- For DAC+: **dtoverlay=hifiberry-dacplus**

When installing the Phoniebox you should
1. Install a fresh Raspbian OS
2. Modify the settings for the Hifiberry as described in the link above
3. Start **Phoniebox-Installer after** you performed a **succesful speakertest** (see link above)
4. Select **MPD** when you configure the Installer
5. For Audio-Output (Iface) enter **Master**

I performed these steps just some days ago on my Phoniebox (Hifiberry Miniamp) and it worked/still works very well

Best Regards

**Reply by MarkAsRead** [reply]
Thanks for getting back to me, this seems to have resolved my issue. Bookmarking your website. :)
